brev-bestilling-boot/src/main/resources/application-embeddedkafka.yml

```diff
@@ -0,0 +1,9 @@
+server:
+    port: 8081
```

Comment:
Vet ikke hva disse skal være. Har bare satt en midlertidig verdi for å unngå kollisjon